docs/ocis/adr/0001-introduce-accounts-service.md

```diff
@@ -1,4 +1,10 @@
-# 1. Introduce an accounts service
+---
+title: "1. Introduce an accounts service"
+date: 2020-06-15T20:21:00+01:00
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/adr
+geekdocFilePath: 0001-introduce-accounts-service.md
+---
 
 * Status: superseded by [ADR-0003](0003-external-user-management.md) <!-- optional -->
```

Comment:
must be
```
* Status: superseded by [ADR-0003]({{< ref "0003-external-user-management.md" >}})
```

Reply:
hm I may need to document another change to the tooling then ...

Reply:
required more changes, forked https://github.com/butonic/adr-tools and make them understand these links